packages/blade/src/components/Carousel/types.ts

```diff
@@ -110,6 +110,13 @@ type CarouselProps = {
    * Sets the height of the carousel
    */
   height?: BoxProps['height'];
+  /**
+   * should show navigation buttons in mobile
+   *
+   * @default true
+   *
+   */
+  showNavigationButtons?: boolean;
```

Comment:
If set to true, shows the navigation buttons. 

Note: This prop has no effect on desktop, we only hide navigation buttons on mobile.